web/client/resources/js/goconvey.js

```diff
@@ -477,8 +498,16 @@ function loadSettingsFromStorage()
 	convey.uiEffects = uiEffects === "true";
 	enumSel("ui-effects", uiEffects);
 
-	if (notif())
-		$('#toggle-notif').toggleClass("fa-bell-o fa-bell " + convey.layout.selClass);
+	enumSel("notification", notif() ? 'on' : 'off');
```

Comment:
would it be possible to make this actually a boolean? The 'on' and 'off' strings everywhere give me the heebie jeebies.